services/movie_session.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
```

Comment:
Add a blank line between different import groups to follow PEP-8. Apply this change to the other places as well